Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -98,6 +98,7 @@ public override void Update()
             if (profile.SimulateEyePosition)
             {
                 InputSystem?.EyeGazeProvider?.UpdateEyeGaze(this, new Ray(CameraCache.Main.transform.position, CameraCache.Main.transform.forward), DateTime.UtcNow);
+                InputSystem?.EyeGazeProvider?.UpdateEyeTrackingStatus(this, true);
```

Comment:
Nit: May as well move this to the line above so that our simulation and actual provider can be consistent (i.e. eye tracking status is always updated first, then eye gaze direction/position).

End effect honestly shouldn't be anything but yeah

Reply:
Done.